enhancements/microshift/microshift-supportability-tools.md

```diff
@@ -0,0 +1,353 @@
+---
+title: microshift-supportability-tools
+authors:
+  - pacevedom
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@pmtk, MicroShift contributor"
+  - "@TurboTurtle, SOS contributor"
+  - "@jrthms, product experience liaison"
+approvers:
+  - "@dhellmann"
+api-approvers:
+  - None
+creation-date: 2023-01-17
+last-updated: 2023-02-06
+tracking-link:
+  - https://issues.redhat.com/browse/USHIFT-389
+see-also:
+  - "/enhancements/oc/must-gather.md"
+---
+
+# MicroShift supportability tools
+## Summary
+This enhancement proposes the addition of supportability tooling to MicroShift.
+
+To debug issues in the cluster we need a single command that any user or
+administrator is able to run to gather all the required information to diagnose
+and solve problems.
+
+## Motivation
+When it comes to supportability, having a tool to extract all relevant
+information from a cluster becomes paramount to solve issues.
+
+Most readers will be familiar with `must-gather` for OpenShift clusters.
+MicroShift does not have such a tool as of today, and it becomes increasingly
+important as it approaches general availability.
+
+The ultimate goal is to have a single command to produce a full report of the
+cluster status.
+
+### User Stories
+
+* As a MicroShift administrator, I want to be able to get a report about current
+  cluster status.
+* As a MicroShift developer, I want to be able to diagnose issues with a detailed
```

Comment:
We usually use "Application Deployer" for the persona you're describing.

For this story, I think a persona from the CEE team is more appropriate, since we're building to their requirements.